### PR TITLE
Fixed NeoAvatar size bug

### LINKED ISSUE #8392: Neo avatar images not properly squared
### Is there an existing issue?

- [X] I have searched the existing issues

### What happened?

image

### Please reproduce in steps

https://canary.kodadot.xyz/ksm/u/FSMdmCLv5gFZ87YerB3M8ZSQ58BGHdSnreC4J6znWpqPQK9?tab=activity&buy=true&sale=true&mint=true&transfer=true&list=true

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

a

### Relevant log output

_No response_

### Payment link for reward

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by shashkovdanil**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8392 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at e0f22ca</samp>

Updated the `ImageMedia` and `NeoAvatar` components to use a new prop for customizing the image component. This improves the consistency and flexibility of the image component across the UI.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at e0f22ca</samp>

> _`NeoAvatar` rises from the ashes_
> _With `image-component-props` it smashes_
> _The old constraints of `width` and `height`_
> _And `fit` as `cover` gives it might_

### CODE REVIEW

File: libs/ui/src/components/MediaItem/type/ImageMedia.vue [hunk 1]
```diff
@@ -35,13 +35,18 @@
 
 <script lang="ts" setup>
 import consola from 'consola'
```
Comment (roiLeo) [inline libs/ui/src/components/MediaItem/type/ImageMedia.vue:37]: oh.. this shouldn't be here

**Review by roiLeo — APPROVED** [review]
looks complex but if it work let's go

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 5.55 USD/DOT ~ 5.405 $DOT
🧗 [16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP](https://polkadot.subscan.io/account/16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP)
🔗 [0x6f75172463b362ea62e901de8a67d981a56bd276b15905ce6c39e9a0c013e1bb](https://polkadot.subscan.io/extrinsic/0x6f75172463b362ea62e901de8a67d981a56bd276b15905ce6c39e9a0c013e1bb)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**